coap/Kconfig

```diff
@@ -71,7 +71,7 @@ menu "CoAP Configuration"
         default 5 if COAP_LOG_NOTICE
         default 6 if COAP_LOG_INFO
         default 7 if COAP_LOG_DEBUG
-        default 9 if COAP_LOG_MBEDTLS
+        default 8 if COAP_LOG_OSCORE
```

Comment:
`CONFIG_LOG_MBEDTLS` was not used anywhere and hence removed?

Reply:
For libcoap, level 9 would enable logging generated by the TLS library - in your case this was MbedTLS. levels 10, 11, 12 etc. gave more verbose TLS logging. A 4.3.2 update is to separate out logging in libcoap between libcoap logging and TLS logging, giving the ability to control the level of the detail of both.  At some point, the libcoap port for esp-idf handed over the logging levels to that specified under the MbedTLS configuration.  So COAP_LOG_MBEDTLS should have been removed then as it had no significance.